src/themes/components/accordion.ts

```diff
@@ -1,45 +1,75 @@
-import {
-  createMultiStyleConfigHelpers,
-  defineStyle,
-} from '@chakra-ui/styled-system';
+import { SystemStyleObject } from '@chakra-ui/styled-system';
 import { accordionAnatomy as parts } from '@chakra-ui/anatomy';
 
-const { definePartsStyle, defineMultiStyleConfig } =
-  createMultiStyleConfigHelpers(parts.keys);
-
-const baseStyleContainer = defineStyle({
-  color: 'gray.900',
-  borderTop: '1px',
-  borderColor: 'gray.200',
-  _last: {
-    borderBottom: '1px',
+const variantLight: Record<string, SystemStyleObject> = {
+  button: {
+    _hover: {
+      bg: 'gray.50',
+    },
+  },
+  container: {
+    color: 'gray.900',
     borderColor: 'gray.200',
+    _last: {
+      borderColor: 'gray.200',
+    },
+  },
+};
+
+const variantDark: Record<string, SystemStyleObject> = {
+  button: {
+    _hover: {
+      bg: 'black',
```

Comment:
Black might be a bit heavy maybe we can use a lighter colour like grey.700 for the hover or there is a darken mechanism

Reply:
Changed to lighter for now